qhub/cli/cost.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from qhub.cost import infracost_report
+
+logger = logging.getLogger(__name__)
+
+
+def create_cost_subcommand(subparser):
```

Comment:
Could you add a little more info to the `help` command regarding what the difference `Resource Breakdown` categories means?

Reply:
Any updates?